pkg/providers/amazon/objectstore/objectstore.go

```diff
@@ -178,7 +179,15 @@ func (s *objectStore) DeleteBucket(bucketName string) error {
 		Bucket: aws.String(bucketName),
 	}
 
-	_, err := s.client.DeleteBucket(input)
+	obj, err := s.ListObjects(bucketName)
+	if err != nil {
```

Comment:
use emperror's WrapWith insted; the woding should be "faieled to list objects" (to be consistent across providers)